dishRoute/dishes-model.js

```diff
@@ -0,0 +1,52 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile.js');
+
+const db = knex(knexConfig.development);
+
+module.exports = {
+    getDishes, 
+    addDish,
+    getDish,
+    update,
+    remove,
+    getRecipes,
+    addRecipe
+};
+
+function getDishes() {
+    return db('dishes');
+};
+
+function getDish(id){
+    return db('recipes')
+    .join('dishes', 'dishes.id', 'recipes.dish_id')
```

Comment:
Since there's no need for a dishes_recipes table I think this is probably better without a join. You could use async/await or .then() and another function (or just write it in here). But the idea is to make our response look like this 
```
dish: {
    id: 1,
    dish_name: "taco",
    recipes: [
        {
            recipe_id: 2, 
           recipe_name: "pork street taco"
        },
        {other recipes....}
    ]
}
```
However you make that happen is up to you